src/components/Scoremeter/index.tsx

```diff
@@ -49,9 +49,9 @@ const ScoreMeter = (props: ScoremeterProps) => {
     }, []);
 
     React.useEffect(() => {
-        let timerId;
+        let timerId: number;
         if (score !== sanitizedReading) {
-            timerId = setTimeout(() => {
+            timerId = window.setTimeout(() => {
```

Comment:
But setTimeout id browser API right? we can use it directly too?  what is the catch here?

Reply:
After making the `timerId` type number, typescript expects that `setTimeout` returns a `Timeout` object as in Node. In order to let typescript know that we are using `setTimeout` from browser API,  added **window** before setTimeout.
https://stackoverflow.com/a/55550147/17948948

The other solution would be adding ''type" with an empty array in `tsconfig.json`
```js
{
  "compilerOptions": {
    "types": []
  }
}
```
https://stackoverflow.com/a/70856713/17948948